### PR TITLE
bugfix/12063-packedbubble-bad-positions-after-width-update

### LINKED ISSUE #12063: Bubbles get stuck in packed bubble chart
#### Expected behaviour
All chart bubbles should be moving along with outer big bubble in UI.

#### Actual behaviour
This UI issue is random in nature.
image

#### Product version

Highcharts version 7.2.0

### PR DESCRIPTION
**Pull request by hubertkozik**
Fixed #12063, packed bubble parent nodes had a bad position after width update.

### CODE REVIEW

File: ts/Series/PackedBubble/PackedBubbleSeries.ts [hunk 1]
```diff
@@ -539,35 +539,40 @@ class PackedBubbleSeries extends BubbleSeries {
...
-            height: (this.parentNodeRadius as any) * 2
-        }, parentOptions);
-        if (!(this.parentNode as any).graphic) {
-            this.graph = (this.parentNode as any).graphic =
-                chart.renderer.symbol((parentOptions as any).symbol)
-                    .add(this.parentNodesGroup);
+        if (
+            this.parentNode &&
+            this.parentNode.plotX &&
+            this.parentNode.plotY &&
```
Comment (pawelfus) [inline ts/Series/PackedBubble/PackedBubbleSeries.ts:558]: When checking the number-type value, always use `H.defined()`, so `0`-values are correctly handled. For example `plotX = 0` and `plotY = 0` (top left corner) would be skipped forever if that happens (unlikely, but still)

**Review by pawelfus — CHANGES_REQUESTED** [review]
Nice, minor change only

**Review by pawelfus — APPROVED** [review]
Thanks!

**Review by TorsteinHonsi — APPROVED** [review]
Thanks!

**Pull request merged**